September Changelog (#2760)
gitpod-io/website: src/lib/contents/changelog/2022-09-30.md

---
title: September Gitpod Release 2022
excerpt:
date: 2022-09-30
# TODO: Update image
image: 2022-09-30.png
alt:
---

<script>
  import Contributors from "$lib/components/changelog/contributors.svelte";
</script>

Welcome to the September edition of the Gitpod Changelog.

**Key highlights**

- [JetBrains Backend Control Center - Common Actions](#jetbrains-backend-control-center---common-actions)
- [JetBrains Backend Control Center - Workspace Classes](#jetbrains-backend-control-center---workspace-classes)
- [JetBrains Reconnect With Open Windows](#jetbrains-reconnect-with-open-windows)
- [New Ports View in VS Code](#new-ports-view-in-vs-code)
- [SSH Gateway default for Self-Hosted VS Code Desktop Users](#ssh-gateway-default-for-self-hosted-vs-code-desktop-users)
- [Fixes and improvements](#fixes-and-improvements)

<!-- TODO: Any insiders notes to highlight? -->

> ### JetBrains Backend Control Center - Common Actions

Reviewer: I wonder whether we could find better name than `Common`? Maybe `Gitpod Workspace Actions`?

1. [Open the Gitpod Dashboard](https://gitpod.io/)
2. [Open your workspace context](/docs/context-urls)
3. [Open your user settings](https://gitpod.io/settings)
4. [Manage workspace access control](https://www.gitpod.io/docs/integrations)
5. [Go to the Gitpod documentation](https://gitpod.io/docs)
6. [Report an issue on GitHub](https://github.com/gitpod-io/gitpod/issues/new/choose)
7. [Follow Gitpod on Twitter](https://twitter.com/gitpod)
8. [Access our community server on Discord](https://www.gitpod.io/chat)
9. [Upgrade your Gitpod subscription](https://www.gitpod.io/pricing)
10. [Stop your workspace](/docs/life-of-workspace#timeouts)
11. [Extend your workspace timeout](/docs/life-of-workspace#timeouts)
12. [Share your a snapshot of your workspace](/docs/sharing-and-collaboration#sharing-snapshots)

All from the comfort of your IDE.

### JetBrains Backend Control Center - Workspace Classes

<!-- TODO: Add more zoomed in image that "points to the class information -->
<!-- TODO: Add something about node vs workspace performance -->

![](https://user-images.githubusercontent.com/2318450/188240455-e5cfba19-5c65-493a-b8c0-0e63c34fd17b.png)

> Did you know that when you're working with JetBrains IDEs in Gitpod you can [see real-time the performance of your workspace](/docs/ides-and-editors/intellij#workspace-performance) via the JetBrains Backend Control Center? This feature is useful when your workspace isn't performing as you'd expect and you want to gather more information. In addition to our recent additions of `Workspace CPU` and `Workspace Memory` we've now added the actual workspace class to the information shown in the control center.

Reviewer 2: ```suggestion
Did you know that when you're working with JetBrains IDEs in Gitpod you can [monitor the performance of your workspace in real-time](/docs/ides-and-editors/intellij#workspace-performance) via the JetBrains Backend Control Center? This feature is useful when your workspace isn't performing as you'd expect and you want to gather more information. In addition to our recent additions of `Workspace CPU` and `Workspace Memory` we've now added the actual workspace class to the information shown in the control center.
```

> ... but why? When you work in a cloud developer environment, you should have the flexibility to choose the right performance profile for your project. Recently we've been working to bring to Gitpod a feature we call "Workspaces Classes" [#8261](https://github.com/gitpod-io/gitpod/issues/8261). With Workspaces Classes you'll be able to customize and configure the specification of your workspace. We know that for larger projects, especially large Java projects using IntelliJ require more resources than the current Gitpod workspaces are designed for.

Reviewer 2: ```suggestion
... but why? When you work in a cloud developer environment, you should have the flexibility to choose the right performance profile for your project. Recently we've been working to bring to Gitpod a feature we call "Workspaces Classes" [#8261](https://github.com/gitpod-io/gitpod/issues/8261). With Workspaces Classes you'll be able to select workspace specifications that better match your use case. For example, you will be able to get more powerful workspaces to run larger Java projects and IntelliJ.
```

### New Ports View in VS Code

<!-- TODO: Add more zoomed in image -->

![](/images/changelog/2022-08-01/ports.png)

You might have seen in our last changelog that [we mentioned a new ports view](https://www.gitpod.io/changelog/new-ports-explorer-changelog-view-and-connection-improvements-for-vs-code#new-ports-explorer-in-vs-code-browser) being added to VS Code (in the browser). At that time we were rolling out the feature to only subsets of our Gitpod users to see how users responded. However, after some good feedback from you (our users) we have pushed the feature out to all users! 🥳

We introduced the new ports view as we wanted to align our experiences on VS Code Desktop with VS Code Browser. We know lots of users like to use Gitpod in the browser sometimes for "smaller changes" like reviewing pull requests. But for more "deep work" many users opt for VS Code Desktop. However and wherever you choose to code, Gitpod will make that as much a comfortable and familiar experience as you're used to.

### SSH Gateway default for Self-Hosted VS Code Desktop Users

<!-- TODO: Add content -->

### Fixes and improvements

> - [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />

Reviewer 2: @loujaybee order of contributors seems borked 🤔 Not problematic, but something to note

Reviewer 3: @atduarte from what I saw it's just ordered alphabetically, I think this is fine as long as the actual PR author is the first one unrelated to their alphabetical order in the list, will incorporate that into https://github.com/gitpod-io/website/pull/2763 🧡.

> - [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
> - [#13203](https://github.com/gitpod-io/gitpod/pull/13203) - [server]: Add a liveness probe which fails when the nodejs event loop lag exceeds a certain threshold <Contributors usernames="andrew-farries,easyCZ" />
> - [#13080](https://github.com/gitpod-io/gitpod/pull/13080) - JetBrains Gateway: Avoid 30 seconds delay when connecting to a workspace using an expired token <Contributors usernames="andreafalzetti,felladrin" />
> - [#13161](https://github.com/gitpod-io/gitpod/pull/13161) - Admin dashboard now has pagination <Contributors usernames="easyCZ,geropl,svenefftinge" />
> - [#13108](https://github.com/gitpod-io/gitpod/pull/13108) - Fix reading .gitpod.yml for self-managed GHE instances. <Contributors usernames="AlexTugarev,geropl" />
> - [#13065](https://github.com/gitpod-io/gitpod/pull/13065) - Added a usage view that displays past workspace sessions on individual and team accounts. <Contributors usernames="easyCZ,jldec,svenefftinge" />
> - [#13084](https://github.com/gitpod-io/gitpod/pull/13084) - Update GoLand IDE image to version 222.4167.25. <Contributors usernames="felladrin" />
> - [#13057](https://github.com/gitpod-io/gitpod/pull/13057) - improved automated code configuration service for `go` <Contributors usernames="Siddhant-K-code,svenefftinge" />
> - [#13001](https://github.com/gitpod-io/gitpod/pull/13001) - Update GitLab API library, which fixes paginated API requests. <Contributors usernames="AlexTugarev,gtsiolis,metcalfc,svenefftinge" />
> - [#13033](https://github.com/gitpod-io/gitpod/pull/13033) - Update PhpStorm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
> - [#13032](https://github.com/gitpod-io/gitpod/pull/13032) - Update PyCharm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
> - [#12670](https://github.com/gitpod-io/gitpod/pull/12670) - Dismiss Usage Limit notifications automatically. <Contributors usernames="AlexTugarev,geropl,jldec,svenefftinge" />
> - [#12994](https://github.com/gitpod-io/gitpod/pull/12994) - Update IntelliJ IDEA IDE image to version 222.4167.29. <Contributors usernames="felladrin" />
> - [#12412](https://github.com/gitpod-io/gitpod/pull/12412) - Add beta notice and label on usage <Contributors usernames="easyCZ,filiptronicek,gtsiolis,laushinka" />
> - [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
> - [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
> - [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
> - [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
> - [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
> - [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
> - [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />

Reviewer 3: Here are the latest updates (cc @loujaybee):
```suggestion
- [#13322](https://github.com/gitpod-io/gitpod/pull/13322) - Fix issue that sporadically prevented workspaces to be opened from a GitHub issue <Contributors usernames="aledbf,atduarte,jenting,utam0k" />
- [#13144](https://github.com/gitpod-io/gitpod/pull/13144) - Enable configuration of core dump behavior in a workspace <Contributors usernames="Furisto,aledbf,csweichel,david-bakin,svenefftinge,utam0k" />
- [#13053](https://github.com/gitpod-io/gitpod/pull/13053) - Do not check all existing git branches during the initial checkout <Contributors usernames="aledbf,jenting,kylos101,sagor999,utam0k" />
- [#13205](https://github.com/gitpod-io/gitpod/pull/13205) - JetBrains: VM options to install plugins <Contributors usernames="akosyakov,andreafalzetti,filiptronicek,subinamathew" />
- [#12445](https://github.com/gitpod-io/gitpod/pull/12445) - Support for storing VS Code edit sessions in sync server <Contributors usernames="akosyakov,andrew-farries,filiptronicek,jeanp413,mustard-mh" />
- [#13295](https://github.com/gitpod-io/gitpod/pull/13295) - Avoid second prebuild been triggered on same commit. <Contributors usernames="AlexTugarev,geropl" />
- [#13296](https://github.com/gitpod-io/gitpod/pull/13296) - Fail prebuild if image build fails. <Contributors usernames="AlexTugarev,easyCZ" />
- [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
- [#13203](https://github.com/gitpod-io/gitpod/pull/13203) - [server]: Add a liveness probe which fails when the nodejs event loop lag exceeds a certain threshold <Contributors usernames="andrew-farries,easyCZ" />
- [#13080](https://github.com/gitpod-io/gitpod/pull/13080) - JetBrains Gateway: Avoid 30 seconds delay when connecting to a workspace using an expired token <Contributors usernames="andreafalzetti,felladrin" />
- [#13161](https://github.com/gitpod-io/gitpod/pull/13161) - Admin dashboard now has pagination <Contributors usernames="easyCZ,geropl,svenefftinge" />
- [#13108](https://github.com/gitpod-io/gitpod/pull/13108) - Fix reading .gitpod.yml for self-managed GHE instances. <Contributors usernames="AlexTugarev,geropl" />
- [#13065](https://github.com/gitpod-io/gitpod/pull/13065) - Added a usage view that displays past workspace sessions on individual and team accounts. <Contributors usernames="easyCZ,jldec,svenefftinge" />
- [#13084](https://github.com/gitpod-io/gitpod/pull/13084) - Update GoLand IDE image to version 222.4167.25. <Contributors usernames="felladrin" />
- [#13057](https://github.com/gitpod-io/gitpod/pull/13057) - improved automated code configuration service for `go` <Contributors usernames="Siddhant-K-code,svenefftinge" />
- [#13001](https://github.com/gitpod-io/gitpod/pull/13001) - Update GitLab API library, which fixes paginated API requests. <Contributors usernames="AlexTugarev,gtsiolis,metcalfc,svenefftinge" />
- [#13033](https://github.com/gitpod-io/gitpod/pull/13033) - Update PhpStorm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#13032](https://github.com/gitpod-io/gitpod/pull/13032) - Update PyCharm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#12670](https://github.com/gitpod-io/gitpod/pull/12670) - Dismiss Usage Limit notifications automatically. <Contributors usernames="AlexTugarev,geropl,jldec,svenefftinge" />
- [#12994](https://github.com/gitpod-io/gitpod/pull/12994) - Update IntelliJ IDEA IDE image to version 222.4167.29. <Contributors usernames="felladrin" />
- [#12412](https://github.com/gitpod-io/gitpod/pull/12412) - Add beta notice and label on usage <Contributors usernames="easyCZ,filiptronicek,gtsiolis,laushinka" />
- [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
- [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
```

P.S.: I removed the long release note from https://github.com/gitpod-io/gitpod/pull/13205 - I think before we figure out the multiline changelog entries, we should add these manually to their manually curated subsections.

---
title: September Gitpod Release 2022

> excerpt:

Reviewer 4: Fill `excerpt` to resolve [this](https://github.com/gitpod-io/website/pull/2760#issuecomment-1263723240)

> excerpt: The latest Gitpod updates for September 2022, including improvements for JetBrains, improving the startup times for large repos and lots of other fixes and improvements.

Reviewer 5: I added an excerpt here @atduarte, feel free to adjust it!

Reviewer 2: Thank you, @Cahllagerfeld! Looks great

---
title: September Gitpod Release 2022
excerpt:
date: 2022-09-30
# TODO: Update image

> image: 2022-09-30.png

Reviewer 2: Colors seem muted. Particularly the Gitpod logo. Is this on purpose?

Author: I'm a little confused, I thought we would swap for [this image](https://github.com/gitpod-io/website/pull/2760#issuecomment-1263531785). The one here is just a placeholder copied from the previous changelog earlier on so the build would pass. 

See also: https://github.com/gitpod-io/website/pull/2760#discussion_r984565307

> ---
> title: September Gitpod Release 2022
> excerpt:
> date: 2022-09-30
> # TODO: Update image
> image: 2022-09-30.png
> ogImage: 2022-09-30/open-graph.jpg
> alt:
> ---

Author: We should delete the old copied image: `static/images/changelog/2022-09-30.png` also. 

This open graph image is not showing for me? @jacobparis 

```suggestion
---
title: September Gitpod Release 2022
excerpt:
date: 2022-09-30
image: 2022-09-30/open-graph.jpg
alt:
---
```

Reviewer 5: It's not on gitpod.io yet. There's a bug with sveltest that we can't display it dynamically. I tested it in my preview though.

Reviewer 5: Do we want to put the OpenGraph-Image (or the same style of image) as the header image?

Reviewer 6: No the open graph image is distinct from the header image

Reviewer 6: > There's a bug with Sveltekit that we can't display it dynamically. I tested it in my preview though.

Not a bug, we just can't statically generate pages that include URLs before we know what URL we're going to be deploying it to. Serverside rendering is ideal here

---
title: September Gitpod Release 2022
excerpt:
date: 2022-09-30
# TODO: Update image
image: 2022-09-30.png
ogImage: 2022-09-30/open-graph.jpg
alt:
---

<script>
  import Contributors from "$lib/components/changelog/contributors.svelte";
</script>

Welcome to the September edition of the Gitpod Changelog!

**Key highlights**

- [JetBrains Backend Control Center - Gitpod workspace actions](#jetbrains-backend-control-center---gitpod-workspace-actions)
- [JetBrains Backend Control Center - Workspace Classes](#jetbrains-backend-control-center---workspace-classes)
- [JetBrains Reconnect With Open Windows](#jetbrains-reconnect-with-open-windows)
- [New Ports View in VS Code](#new-ports-view-in-vs-code)
- [SSH Gateway default for Self-Hosted VS Code Desktop Users](#ssh-gateway-default-for-self-hosted-vs-code-desktop-users)
- [Configurable core dump behavior](#configurable-core-dump-behavior)
- [Faster startup times for large repositories](#faster-startup-times-for-large-repositories)
- [Fixes and improvements](#fixes-and-improvements)

> <!-- TODO: Any insiders notes to highlight? -->

Author: ```suggestion
```

---
title:
excerpt:
date: 2022-09-30
image: 2022-09-30.jpg
alt:
---

<script>
  import Contributors from "$lib/components/changelog/contributors.svelte";
</script>

TODO: Insert the featured changelog entry here.

<p><Contributors usernames="" /></p>

### Fixes and improvements

> - [#12202](https://github.com/gitpod-io/gitpod/pull/12202) - [installer]: move the kots install script into a bash file in Installer <Contributors usernames="MrSimonEmms,Pothulapati,corneliusludmann" />
> - [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center  <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
> - [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace  <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
> - [#12577](https://github.com/gitpod-io/gitpod/pull/12577) - [single-cluster/eks] Add `cluster-autoscaler` <Contributors usernames="Pothulapati,nandajavarma" />
> - [#12604](https://github.com/gitpod-io/gitpod/pull/12604) - Update stable VSCode Browser to `1.71` <Contributors usernames="iQQBot,mustard-mh" />
> - [#12445](https://github.com/gitpod-io/gitpod/pull/12445) - [server] Support for storing vscode edit sessions in sync server <Contributors usernames="akosyakov,andrew-farries,filiptronicek,jeanp413,mustard-mh" />
> - [#12496](https://github.com/gitpod-io/gitpod/pull/12496) - [local-preview] Remove unused dead code <Contributors usernames="MrSimonEmms,Pothulapati" />
> - [#12657](https://github.com/gitpod-io/gitpod/pull/12657) - [local-preview] set low memory & cpu requirements <Contributors usernames="Pothulapati,nandajavarma" />
> - [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
> - [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
> - [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
> - [#12683](https://github.com/gitpod-io/gitpod/pull/12683) - [installer]: config detects shiftfs and containerd values <Contributors usernames="MrSimonEmms,nandajavarma,sagor999" />
> - [#12679](https://github.com/gitpod-io/gitpod/pull/12679) - [installer]: create config command and deprecate init command <Contributors usernames="ArthurSens,MrSimonEmms,mads-hartmann,nandajavarma" />
> - [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />

Reviewer 3: ```suggestion
- [#13186](https://github.com/gitpod-io/gitpod/pull/13186) - Restrict reuse of phone numbers for verification <Contributors usernames="atduarte,easyCZ,svenefftinge" />
- [#13203](https://github.com/gitpod-io/gitpod/pull/13203) - [server]: Add a liveness probe which fails when the nodejs event loop lag exceeds a certain threshold <Contributors usernames="andrew-farries,easyCZ" />
- [#13080](https://github.com/gitpod-io/gitpod/pull/13080) - JetBrains Gateway: Avoid 30 seconds delay when connecting to a workspace using an expired token <Contributors usernames="andreafalzetti,felladrin" />
- [#13161](https://github.com/gitpod-io/gitpod/pull/13161) - Admin dashboard now has pagination <Contributors usernames="easyCZ,geropl,svenefftinge" />
- [#13108](https://github.com/gitpod-io/gitpod/pull/13108) - Fix reading .gitpod.yml for self-managed GHE instances. <Contributors usernames="AlexTugarev,geropl" />
- [#13065](https://github.com/gitpod-io/gitpod/pull/13065) - Added a usage view that displays past workspace sessions on individual and team accounts. <Contributors usernames="easyCZ,jldec,svenefftinge" />
- [#13084](https://github.com/gitpod-io/gitpod/pull/13084) - Update GoLand IDE image to version 222.4167.25. <Contributors usernames="felladrin" />
- [#13057](https://github.com/gitpod-io/gitpod/pull/13057) - improved automated code configuration service for `go` <Contributors usernames="Siddhant-K-code,svenefftinge" />
- [#13001](https://github.com/gitpod-io/gitpod/pull/13001) - Update GitLab API library, which fixes paginated API requests. <Contributors usernames="AlexTugarev,gtsiolis,metcalfc,svenefftinge" />
- [#13033](https://github.com/gitpod-io/gitpod/pull/13033) - Update PhpStorm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#13032](https://github.com/gitpod-io/gitpod/pull/13032) - Update PyCharm IDE image to version 222.4167.33. <Contributors usernames="felladrin" />
- [#12670](https://github.com/gitpod-io/gitpod/pull/12670) - Dismiss Usage Limit notifications automatically. <Contributors usernames="AlexTugarev,geropl,jldec,svenefftinge" />
- [#12994](https://github.com/gitpod-io/gitpod/pull/12994) - Update IntelliJ IDEA IDE image to version 222.4167.29. <Contributors usernames="felladrin" />
- [#12412](https://github.com/gitpod-io/gitpod/pull/12412) - Add beta notice and label on usage <Contributors usernames="easyCZ,filiptronicek,gtsiolis,laushinka" />
- [#12852](https://github.com/gitpod-io/gitpod/pull/12852) - Add link to support on phone verification <Contributors usernames="gtsiolis,svenefftinge" />
- [#12833](https://github.com/gitpod-io/gitpod/pull/12833) - Improve fetching repositories loading state <Contributors usernames="gtsiolis,svenefftinge" />
- [#12771](https://github.com/gitpod-io/gitpod/pull/12771) - Show personal Usage page. <Contributors usernames="AlexTugarev,andrew-farries" />
- [#12621](https://github.com/gitpod-io/gitpod/pull/12621) - JetBrains IDEs now have actions related to Gitpod, which can be accessed via Control Center and via the Search Menu. <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12669](https://github.com/gitpod-io/gitpod/pull/12669) - Reword "Spending Limit" to "Usage Limit" <Contributors usernames="AlexTugarev,andrew-farries,jldec" />
- [#12568](https://github.com/gitpod-io/gitpod/pull/12568) - JetBrains: Provide workspace class info in Backend Control Center <Contributors usernames="akosyakov,andreafalzetti,felladrin" />
- [#12232](https://github.com/gitpod-io/gitpod/pull/12232) - Fixed JetBrains connection loop when connecting twice to the same workspace <Contributors usernames="akosyakov,andreafalzetti,felladrin" />

```

Author: Update based on: https://github.com/gitpod-io/website/pull/2759/files adjusted script
